### DISCUSSION TITLE
Is it possible to use date versioning instead?

### DISCUSSION THREAD

**Discussion by anthonyma94** [semantic-release/semantic-release#2308] [General] [answered]
I know date versioning doesn't follow semantic versioning rules, but for our project it doesn't make sense to use semantic versions. I like all the other rules of semantic versioning, but is there a way to incorporate the date in the version number?

**Comment by travi** [comment, marked as answer]
a core purpose of semantic-release is making it simple to use semantic versioning correctly. it is not a general workflow tool. date-based versioning is not supported
